tests/sanitizers/test_sanitizers.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+##############################################################
+# Folders structure (11.08.2020)
+# NOTE: Drawing will be removed after approval script
+##############################################################
+# facelift/                 <-- directory with source code
+# ├─ build/
+# │  ├─                     <-- working directory
+# │  └─
+# ├─ ci/
+# │  ├─ appveyor
+# │  └─ travis
+# ├─ cmake/
+# ├─ CMakeLists.txt
+# ├─ src/
+# ├─ tests/
+# │  ├─ asyncfunctions/
+# │  ├─ benchmarking/
+# │  ├─ cmake/
+# │  ├─ CMakeLists.txt
+# │  ├─ combined/
+# │  ├─ sanitizers/
+# │  │   ├─ read.me
+# │  │   └─ test_sanitizers.sh*   <--- executable script
+
+function outerdir(){
+   echo "$1" | sed -E 's@/[^.]+.*@/@'
+}
+DIR_BUILD=$PWD
+DIR_OUTER=$(outerdir "$0")
+if [ "$DIR_OUTER" = "./" ]
+then
+     echo "The script must be run from the 'facelift/build'"
```

Comment:
we do not want that. See my remark on the other PR.

Reply:
Okay.